github-profiles/.vite/deps_temp/package.json

```diff
@@ -0,0 +1 @@
+{"type":"module"}
```

Comment:
I think this folder must be inside your Discord user folder, to avoid conflicts in the merge